src/services/programs.js

```diff
@@ -157,6 +157,29 @@ module.exports = class ProgramsHelper {
 				},
 			}
 
+			if (
```

Comment:
@praveenKDass hence this code is duplicating for program and project please use a generic function also, if u are fetch org details and its not in orgConfit then add that.